includes/LogisticsOrderFactory.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Factory for creating Logistics Orders
+ *
+ * @package GbWeiss
+ */
+
+namespace GbWeiss\includes;
+
+defined('ABSPATH') || exit;
+
+use Towa\GebruederWeissSDK\Model\LogisticsOrder;
+
+/**
+ * Factory for creating Logistics Orders
+ */
+class LogisticsOrderFactory
+{
+    /**
+     * Creates a logistics order from a WooCommerce order
+     *
+     * @param object $wooCommerceOrder The order to be converted into a logistics order.
+     * @return LogisticsOrder
+     */
+    public static function buildFromWooCommerceOrder(object $wooCommerceOrder): LogisticsOrder
+    {
+        return new LogisticsOrder();
```

Comment:
am i blind or do i just not see where the parameter is used here to create the logistics order?

Reply:
Creating Logistics Orders is part of the next PR. Its not in the scope of this subtask.

Reply:
+1